src/modules/session/types.ts

```diff
@@ -14,6 +17,7 @@ export interface Profile {
   readonly pictureUrl: string;
   readonly role: string;
   readonly genre: string;
+  readonly verifiedStatus: "unverified" | "pending" | "verified";
```

Comment:
I've thought about this a little more, FE should never update the verifiedStatus to "verified". Should we create 2 fields instead? @scandycuz thoughts?
`readonly isVerified: boolean;`
`verifiedStatus: "unverified" | "pending";`

Reply:
You are correct, that change will be reflected in the next push